Lib/test/test_embed.py

```diff
@@ -706,8 +703,6 @@ def check_path_config(self, configs, expected):
         expected['module_search_path'] = os.path.pathsep.join(config['module_search_paths'])
         expected['program_full_path'] = config['executable']
 
-        self.assertEqual(configs['path_config'], expected)
```

Comment:
So we're not actually checking the path config now, as the method name implies, right?